### DISCUSSION TITLE
使用Github Actions编译alist-web前端时提示缺少api_token和project_id

### DISCUSSION THREAD

**Discussion by Simonzhang8** [AlistGo/alist#6751] [Q&A]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
我已经阅读了[文档](https://alist.nn.ci)。

- [X] I'm sure there are no duplicate issues or discussions.
我确定没有重复的issue或讨论。

- [X] I'm sure it's due to `AList` and not something else(such as [Network](https://alist.nn.ci/faq/howto.html#tls-handshake-timeout-read-connection-reset-by-peer-dns-lookup-failed-connect-connection-refused-client-timeout-exceeded-while-awaiting-headers-no-such-host) ,`Dependencies` or `Operational`).
我确定是`AList`的问题，而不是其他原因（例如[网络](https://alist.nn.ci/zh/faq/howto.html#tls-handshake-timeout-read-connection-reset-by-peer-dns-lookup-failed-connect-connection-refused-client-timeout-exceeded-while-awaiting-headers-no-such-host)，`依赖`或`操作`）。

- [X] I'm sure this issue is not fixed in the latest version.
我确定这个问题在最新版本中没有被修复。

### AList Version / AList 版本

v3.35.0

### Driver used / 使用的存储驱动

无

### Describe the bug / 问题描述

使用Github Actions编译alist-web前端时提示缺少api_token和project_id，但好像也没有地方需要填写，而且就算要填，填的是什么？
![image](https://github.com/alist-org/alist/assets/165749751/1caafe6f-2acf-454e-874d-00127ac8220f)

### Reproduction / 复现链接

https://github.com/Simonzhang8/alist-web-zxm/actions/runs/9867467723/job/27247889726

### Config / 配置

https://github.com/Simonzhang8/alist-web-zxm/tree/main/.github/workflows

**Comment by li-peifeng** [comment]
在你的repo设置里添加

**Comment by Simonzhang8** [comment]
> 在你的repo设置里添加

想问一下repo设置在哪里啊？

**Comment by li-peifeng** [comment]
![IMG_0043](https://github.com/alist-org/alist/assets/103113557/3f49bcb0-783b-4b34-b0d5-66bd1f6d5109)

**Comment by Simonzhang8** [comment]
> ![IMG_0043](https://private-user-images.githubusercontent.com/103113557/347828567-3f49bcb0-783b-4b34-b0d5-66bd1f6d5109.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3MDM2OTUsIm5iZiI6MTcyMDcwMzM5NSwicGF0aCI6Ii8xMDMxMTM1NTcvMzQ3ODI4NTY3LTNmNDliY2IwLTc4M2ItNGIzNC1iMGQ1LTY2YmQxZjZkNTEwOS5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTFUMTMwOTU1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmRjZWY4MDFhMGI1ZmE4NDdhNzJiYTVkOTg1ZDEwYjQzYWIxNGVkODI2N2E5MmE0ZDZlNjQxNGYyY2EwNWQ4OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.uQ2E35mxZ9SqCXlL7JicOqjDjWY3YahfEQowFgQ6dYo)

好的好的，但是里面填什么？这个id和token是什么的的id和token？

**Comment by li-peifeng** [comment]
获取CrowdinToken，AList语言包

https://crowdin.com/settings#api-key

CROWDIN_PERSONAL_TOKEN：在上方步骤中获取的Crowdin Token
CROWDIN_PROJECT_ID：AList 在 [Crowdin 项目的ID](https://crowdin.com/project/alist)，ID为 526584

**Comment by Simonzhang8** [comment]
> CROWDIN_PERSONAL_TOKEN：在上方步骤中获取的Crowdin Token CROWDIN_PROJECT_ID：AList 在 [Crowdin 项目的ID](https://crowdin.com/project/alist)，ID为 526584

现在报错提示You do not have permission to view/edit project with provided id.怎么办？
![image](https://github.com/alist-org/alist/assets/165749751/9fdd59b2-363f-4d5c-a504-02dc190d8677)

**Comment by li-peifeng** [comment]
你仔细看一下这个吧

**Comment by xhofe** [comment]
Crowdin 环境变量仅为在CI中获取语言包使用，仅管理员Token有此权限，如果你不需要多语言可以去掉其中的crowdin步骤，如果需要 可以修改workflow文件改为使用wget/curl 下载语言包 而不是使用crowdin
